### DISCUSSION TITLE
Make File command Error

### DISCUSSION THREAD

**Discussion by DevSwayam** [Cyfrin/foundry-full-course-cu#195] [Q&A] [answered]
So i installed Ubantu in my windows and i was going throught ReadMe of lesson 7. On time https://youtu.be/sas02qSFZ74?t=7921 The command or URL of StackOverFLow which has been provided to install Make File disnt work in my case. I did some prompting and i got a command which install the make file on ubantu.

``` Terminal 
sudo apt install make
```

After this Terminal will ask for password and after that package will get installed. 
Can anyone add this in Readme For Ubantu users?

**Comment by PatrickAlphaC** [comment]
Ah thanks for this! Would you like to make the PR to the README?

**Reply by DevSwayam** [reply]
Yea Will Do

**Comment by DevSwayam** [comment, marked as answer]
@PatrickAlphaC sir i did it 
#### Reference = Issue #204  and PR for it #203.

**Reply by PatrickAlphaC** [reply]
Thank you!

**Reply by DevSwayam** [reply]
Anytime sir!!
